pkgs/development/python-modules/ansible-builder/default.nix

```diff
@@ -40,6 +41,7 @@ buildPythonPackage rec {
     jsonschema
     requirements-parser
```

Comment:
Is that even used https://github.com/search?q=repo%3Aansible%2Fansible-builder%20requirements-parser&type=code ?